catch-of-the-day/now.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 2,
+  "name": "cotd",
+  "builds": [{ "src": "package.json", "use": "@now/static-build" }],
+  "routes": [
+    { "src": "^/static/(.*)", "dest": "/static/$1" },
```

Comment:
Looks great, Ronan. I believe we should add a route for images:
`{ "src": "^/images/(.*)", "dest": "/images/$1" },`